doc/migration/version-upgrade-guide.md

```diff
@@ -335,6 +335,144 @@ Upon updating the project to `v0.8.2` the following breaking changes apply:
   replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
   ```
 
+## `v0.10.x`
+
+- The scorecard configuration format for the `operator-sdk scorecard` command has changed. See [`doc/test-framework/scorecard`](../test-framework/scorecard.md) for more info.
+- The CSV config field `role-path` is now `role-paths` and takes a list of strings.
+    Replace:
+    ```yaml
+    role-path: path/to/role.yaml
+    ```
+    with:
+    ```yaml
+    role-paths:
+    - path/to/role.yaml
+    ```
+
+### modules
+
+- Ensure the the following `replace` directives are present in your `go.mod` file:
+    ```
+    replace (
+            github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+            // Pinned to v2.9.2 (kubernetes-1.13.1) so https://proxy.golang.org can
+            // resolve it correctly.
+            github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.0.0-20190424153033-d3245f150225
+            k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
+            sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.12
+            sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
+    )
+
+    replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.10.0
+    ```
+
+## `v0.11.x`
+
+**NOTE:** this version uses Kubernetes v1.14.x and controller-runtime v0.2.x, both of which have breaking API changes. See the [changelog][changelog] for more details.
+
+### Import updates
+
+- Replace import `sigs.k8s.io/controller-runtime/pkg/runtime/scheme` with `sigs.k8s.io/controller-runtime/pkg/scheme` in:
+  - `./pkg/apis/<group>/<version>/register.go`
+- Replace import `sigs.k8s.io/controller-runtime/pkg/runtime/log` with `sigs.k8s.io/controller-runtime/pkg/log` in:
+  - `cmd/manager/main.go`
+  - `./pkg/controller/<kind>/<kind>_controller.go`
+- Replace import `sigs.k8s.io/controller-runtime/pkg/runtime/signals` with `sigs.k8s.io/controller-runtime/pkg/manager/signals` in:
+  - `cmd/manager/main.go`
+
+### controller-runtime API updates
```

Comment:
The following steps are easier to be followed if we update the modules/deps first. 
I'd recommend changing the order.